modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageServiceImpl.java

```diff
@@ -688,6 +689,17 @@ public MBMessageDisplay getMessageDisplay(long messageId, int status)
 			getGuestOrUserId(), messageId, status);
 	}
 
+	@Override
+	public List<MBMessage> getSiteUserMessageBoardMessagesActivity(
+			long userId, long groupId, int start, int end)
+		throws PortalException {
+
+		_userPermission.check(getPermissionChecker(), userId, ActionKeys.VIEW);
+
+		return mbMessageLocalService.getSiteUserMessageBoardMessagesActivity(
+			userId, groupId, start, end);
+	}
+
```

Comment:
Add public int getSiteUserMessageBoardMessagesActivityCount(
 		long groupId, long userId) too to call it on MessageBoardMessageResourceImpl.

Reply:
Remember how you told me maybe it's not necessary? I looked this up and methods that use count in LocalService don't create that method in Service. I talked to you about it @joseabelenda